docs/source/usage/index.rst

```diff
@@ -94,6 +94,24 @@ Users can view available offers and claim an offer to create a lease.
 | Delete Lease | ``openstack esi lease delete <lease_uuid>``                                                |
 +--------------+--------------------------------------------------------------------------------------------+
 
+Subleases
+~~~~~~~~~
+
+A lessee can perform owner actions on a node that they have leased, effectively subleasing a node. Note that a sublessee cannot further sublease a node.
+
+Limited Offers and Subprojects
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+An offer can be made available to a limited number of projects by using the ``--lessee`` option when creating the offer.
+
++----------------------+------------------------------------------------------------------------------------------------------------------------------+
+|                      | **Owner Actions**                                                                                                            |
++----------------------+------------------------------------------------------------------------------------------------------------------------------+
+| Create Limited Offer | ``openstack esi offer create --start-time <start_time> --end-time <end_time> --lessee <lessee_project> <node_uuid_or_name>`` |
++----------------------+------------------------------------------------------------------------------------------------------------------------------+
+
+An offer created in such a way is available not only to the lessee project, but to the entire subtree of projects beneath the lessee project. As a result, using this feature effectively requires projects to be carefully organized.
```

Comment:
Do you think it would be necessary to specify how to do this or would it be safe to assume that readers of this document have a solid grasp on how to work with hierarchical keystone objects?

Reply:
I'm wary of putting keystone-specific documentation in here. Unfortunately there's no upstream documentation specifically around hierarchical keystone projects - just specs and the like - but I'm fine assuming some knowledge of keystone.

Reply:
Sounds good. I've found this: (https://docs.openstack.org/keystone/latest/admin/cli-manage-projects-users-and-roles.html) and this: (https://docs.openstack.org/keystone/latest/admin/cli-manage-projects-users-and-roles.html) but none of those really do a good job explaining hierarchical keystone objects..